torch/csrc/Module.cpp

```diff
@@ -544,12 +544,12 @@ PyObject *THPModule_toDLPack(PyObject *_unused, PyObject *data)
   THPUtils_assert(THPModule_isTensor(data), "data must be a Tensor");
   auto atTensor = torch::createTensor(data);
   DLManagedTensor* dlMTensor = at::toDLPack(atTensor);
-  return PyCapsule_New(dlMTensor, "dltensor", NULL);
+  return PyCapsule_New(dlMTensor, "DLManagedTensor", NULL);
```

Comment:
can I ask why this was needed to be changed from "dltensor"?

Reply:
unless we push the changes to actual dlpack repo, I feel a little reluctant here to embed DLManagedTensor since if the DLPack repo requests changes, then we'd have to circle back to changing this. Rather I felt like keeping it more general, short and concise was fine.

Reply:
The choice of name is significant -- all frameworks that produce or consume the pycapsule need to call it the same thing (if they don't their code will fail to interoperate with this code). So I figure the best name to call it would the C-namespace name of the underlying struct it contains.